main.go

```diff
@@ -390,9 +390,16 @@ func initDaemon(config *menderConfig, dev installer.DualRootfsDevice, env instal
 	mp.dualRootfsDevice = dev
 
 	controller, err := NewMender(config, *mp)
-	if controller == nil {
-		mp.store.Close()
-		return nil, errors.Wrap(err, "error initializing mender controller")
+	if err != nil {
+		// Ignore server certificate error  (See: MEN-2378)
+		if _, ok := err.(*client.ClientServerCertificateError); ok {
+			mp.store.Close()
+			return nil, errors.Wrap(err, "error initializing mender controller")
+		}
+		errstr := "IGNORING ERROR: The client server-certificate can not be loaded error: (%v). The client will " +
+			"continue running, but will not be able to communicate with the server. If this is not your intention " +
+			"please add a valid server certificate"
```

Comment:
The message is very long, and still I think there is something missing.

In my opinion, we should add that the client is going to rollback if this situation persists.

Reply:
Yes, ideally. However this has been discussed here already ;)
https://tracker.mender.io/browse/MEN-2378

Reply:
Right.